pkg/karmadactl/promote.go

```diff
@@ -442,7 +445,7 @@ func createOrUpdateClusterPropagationPolicy(karmadaClient *karmadaclientset.Clie
 
 // buildPropagationPolicy build PropagationPolicy according to resource and cluster
 func buildPropagationPolicy(gvr schema.GroupVersionResource, opts CommandPromoteOption) *policyv1alpha1.PropagationPolicy {
```

Comment:
Besides 
Can you tell me why do not use pointer here  that we can reduce the copy of opts
 like
`func (opts *CommandPromoteOption) buildPropagationPolicy(gvr schema.GroupVersionResource) *policyv1alpha1.PropagationPolicy` 
@lonelyCZ

Reply:
This is probably a lingering habit, if necessary, we can try to clean them.